torch/distributed/fsdp/_common_utils.py

```diff
@@ -351,6 +352,32 @@ def return_fn(param_to_fqns):
     )
 
 
+@no_type_check
+def _log_post_backward_hook(
+    state: _FSDPState, handle: "FlatParamHandle", log: logging.Logger
+) -> None:
+    # Under TORCH_DISTRIBUTED_DEBUG=INFO, log the module names this hook fires for.
+    # Below logging of module names this post-bwd hook fires for can help debug certain
+    # cases where hooks don't fire, such as under certain activation checkpoint configs.
+    if state._use_orig_params and handle._debug_level == dist.DebugLevel.INFO:
+        param_fqns = get_handle_fqns_from_root(state, handle)
+        log.warning("FSDP firing post-backward hooks for parameters %s", param_fqns)
+
+
+@no_type_check
+def get_handle_fqns_from_root(
```

Comment:
small nit: It does not matter, but I wonder if we should just add a prefix `_` on this function name for consistency with the others.